src/itouch.cpp

```diff
@@ -1,4 +1,5 @@
 #include <irods/rodsClient.h>
+#include <irods/rodsError.h>
```

Comment:
Is this needed?

Reply:
Yes. There is a call to `printErrorStack` on line 77.

Reply:
Good.